### ISSUE TITLE
Support for Mac Os X

### ISSUE THREAD

**Issue by andreaschiona** [Siouan/frontend-gradle-plugin#28]
**Is your feature request related to a problem? Please describe.**
Please add the support for the MAC OS X.

**Comment by vboriesazeau** [comment]
Hi Andrea,
Thank you for your feedback. I'm currently fixing some issues with Linux/Mac OS, since I just saw Node distributions layout is not the same as under Windows, and `node` and `npm` executables may not be found.
If you tried the plugin under Mac OS X, what problem did you encounter exactly? I want to be sure it will be fixed with my current work.
Thanks!

**Comment by andreaschiona-tria** [comment]
This is the error:

```
Air-di-Andrea:botteghe andrea$ ./gradlew bootRun
Starting a Gradle Daemon (subsequent builds will be faster)

> Task :installNode FAILED
[installNode] Removing content in install directory '/Sviluppo/botteghe/node'.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':installNode'.
> org.siouan.frontendgradleplugin.core.DistributionUrlResolverException: This platform is not supported yet: Mac OS X, x86_64

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 8s
1 actionable task: 1 executed```

**Comment by vboriesazeau** [comment]
Great! That was exactly what I needed, thank you!

**Comment by vboriesazeau** [comment]
MacOS support may also be broken due to symlinks issue (see #29).

**Comment by vboriesazeau** [comment]
Mac OS/Linux support shall be fixed, please upgrade to release 1.1.2.
